tests/e2e/Homework/AgileWay-BookingFlights.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import  {SignIn} from 'e2e-tests/AgileWay.page.object/signIn.page';
+import { BookingAFlight } from 'e2e-tests/AgileWay.page.object/booking.page';
+
+test.describe('Booking flights on AgileWay website', () => {
+
+    let signIn: SignIn;
+    let bookingFlight: BookingAFlight;
+
+    test.beforeEach(async ({ page }) => {
+        signIn = new SignIn(page);
+        bookingFlight = new BookingAFlight(page);
+    });
+
+  test('Passenger can successfully book a flight with return using a master card', async ({page}) => {
+    await signIn.goTo();
+    await signIn.signInToAgileWay('agileway', 'testW1se');
+    await bookingFlight.selectFlight('return','Sydney', 'New York', '11', 'Feburary 2025', '8:30', '20', 'July 2025');
+    await expect(page.locator('body')).toContainText('2025-07-20');
+    await expect(page.locator('body')).toContainText('2025-02-11');
+    await expect(page.locator('body')).toContainText('Sydney');
+    await expect(page.locator('body')).toContainText('New York');
```

Comment:
here (and in other places),  - expect is not needed, because the main goal of test is that user would be able to book flight using according card. however, if you want to double check if needed data been entered and shown correctly - I would suggest to asserting it in  async selectFlight